uk.co.compucorp.civicrm.hrleaveandabsences/hrleaveandabsences.php

```diff
@@ -370,7 +370,7 @@ function hrleaveandabsences_civicrm_tabset($tabsetName, &$tabs, $context) {
     $contactId = $context['contact_id'];
     $tabs[] = [
       'id'        => 'absence',
-      'url'       => CRM_Utils_System::url('civicrm/contact/view/absence', ['cid' => $contactId]),
+      'url'       => CRM_Utils_System::url('civicrm/contact/view/absence', ['cid' => $contactId], FALSE, '/absence-tab/report'),
```

Comment:
Backend should not know which is the default tab, instead that can be handled by frontend. So here the code should be 
`'url'       => CRM_Utils_System::url('civicrm/contact/view/absence', ['cid' => $contactId], FALSE, '/absence-tab'),`

then in the frontend we can use any of solutions proposed in https://stackoverflow.com/questions/29491079/redirect-a-state-to-default-substate-with-ui-router-in-angularjs

Specially I like this one `As of version 1.0 of the ui-router it supports a redirectTo property.`

Reply:
> Specially I like this one https://stackoverflow.com/a/46439298/2400594

Unfortunately we are not using ui-router v1, we are at v0.2.15